readthedocs/search/tests/utils.py

```diff
@@ -0,0 +1,14 @@
+from readthedocs.search.tests.dummy_data import DUMMY_PAGE_JSON
+
+
+def get_search_query(project_slug, page_num=0, data_type='title'):
+    all_pages = DUMMY_PAGE_JSON[project_slug]
```

Comment:
Could use a docstring and perhaps a better name. Maybe `get_title_of_project_page` or something like this, which specifies a bit more what it's doing.

Reply:
Yeah. I think naming it something like `get_search_query_from_project_file` would be better. because its not only getting the title. Its getting as per the `data_type` provided to the function.
I will add docstring surely.